src/test/java/juja/sqlcmd/DatabaseManagerTest.java

```diff
@@ -1,13 +1,26 @@
 package juja.sqlcmd;
 
+import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
 
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Date;
+
 import static junit.framework.TestCase.assertFalse;
+import static org.junit.Assert.assertArrayEquals;
 import static org.junit.Assert.assertTrue;
 
 public class DatabaseManagerTest {
+    private static final String DB_NAME = "sqlcmd";
+    private static final String USER = "sqlcmd";
+    private static final String PASSWORD = "sqlcmd";
+    private static final String TEST_DB_CONNECTION_URL = "jdbc:postgresql://localhost:5432/";
```

Comment:
Test DB connection URL doesn't contain DB name
May be rename this variable?

Reply:
I will rename this constant